### ISSUE TITLE
[Bug]: Can't send messages through a server console w/ FTB Essentials installed. (loads of errors.)

### ISSUE THREAD

**Issue by Gbergz** [FTBTeam/FTB-Mods-Issues#346]
### Mod

FTB Essentials

### Mod version

ftb-essentials-1900.1.8-build.44

### Forge / Fabric version

1.19 - 41.0.63

### Modpack & version

TNP Limitless 6 v0.6.0

### What issue are you having?

Currently can't send messages through the server console w/ FTB Essentials installed. (loads of errors.)

Tried many commands like "say test" "whisper player <test>", nothing works. Only a bunch of errors.

### Logs

https://gist.github.com/Gbergz/b3465a5312aa504672f5b0181bf60617

### Steps to reproduce

Install a Server
install FTB Essentials
type "say hello" or "whisper player <message>"
loads of errors, message not sent.
profit.

### Anything else to note?

Basically it.
I have tried without the **mc2discord** mod that is mentioned in the stacktrace in the logs, same result. errors.

**Comment by MichaelHillcox** [comment]
This is fixed as of FTB Ranks (1900.1.10-build.43), you can get the indev build here as it's not on curseforge yet :+1:

https://maven.ftb.dev/dev/ftb/mods/ftb-ranks-forge/1900.1.10-build.43.jar

**Comment by MichaelHillcox** [comment]
Released the fix a week or so ago
